examples/arkode/C_serial/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@
 set(ARKODE_examples
     # tests that are always run
     "ark_analytic\;\;"
+    "lsrk_analytic\;\;"
+    "lsrk_analytic_VarJac\;\;"
+    "erk_analytic\;\;"
```

Comment:
Do you really need to keep `erk_analytic` here?  I thought it was just a stepping-stone for building your `lsrk_analytic` example function.